tools/integration_tests/mounting/gcsfuse_test.go

```diff
@@ -230,10 +232,19 @@ func (t *GcsfuseTest) Version() {
 		output, err := cmd.CombinedOutput()
 
 		AssertEq(nil, err)
-		AssertTrue(strings.Contains(string(output), "fake_version"))
+		if setup.TestInstalledPackage() {
+			assertContains("0.0.0", string(output))
+		} else {
+			assertContains("fake_version", string(output))
+		}
 	}
 }
 
+func assertContains(expected, actual string) {
+	if !strings.Contains(actual, expected) {
+		logger.Fatal("Actual: %s does not contain expected: %s", actual, expected)
+	}
+}
```

Comment:
nit: leave empty line after this.
Also did you check once the failure scenario is executing properly?

Reply:
Yes, verified that test failure is captured.